### PR TITLE
add a client side search on the table name in the data selector.

### LINKED ISSUE #1890: Filter tables by name in dropdown selectors
Would be great to make it easier to use with large table collections

### PR DESCRIPTION
**Pull request by agilliland**
resolves #1890

### CODE REVIEW

File: frontend/src/query_builder/AccordianList.jsx [hunk 1]
```diff
@@ -157,9 +165,28 @@ export default class AccordianList extends Component {
                                 }
                             </div>
                         : null }
+
+                        { searchable &&
+                            /* NOTE: much of this structure is here just to match strange stuff in 'List-item' below so things align properly */
+                            <div className="px1 pt1">
```
Comment (kdoh) [inline frontend/src/query_builder/AccordianList.jsx]: I think this is a big enough block of code that it'd be worth factoring out into its own component (maybe a stateless one) like "ListSearchField" or similar  so we prevent copypasta and style drift.

Comment (agilliland) [inline frontend/src/query_builder/AccordianList.jsx]: @kdoh took your suggestion and separated out the relevant parts into their own component.  check it out and let me know if that's what you were thinking.

**Comment by agilliland** [comment]
okay, showed this to @mazameli and he said :+1: , so unless there is anything else from @tlrobinson or @kdoh i'm going to go ahead and merge this.

File: frontend/src/query_builder/AccordianList.jsx (hunk 1 above)
Comment (kdoh) [inline frontend/src/query_builder/AccordianList.jsx]: :+1:

**Pull request merged**